framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -414,170 +444,24 @@ describe('MessageRecoveryCommand', () => {
 			transactionParams.idxs = [1];
 			transactionParams.chainID = chainID;
 
-			commandVerifyContext = createCommandVerifyContext(transaction, transactionParams);
+			commandExecuteContext = createCommandExecuteContext(transaction, transactionParams);
 
 			await interopModule.stores
 				.get(TerminatedOutboxStore)
-				.set(createStoreGetter(commandVerifyContext.stateStore as any), chainID, {
+				.set(createStoreGetter(commandExecuteContext.stateStore as any), chainID, {
 					outboxRoot,
 					outboxSize: terminatedChainOutboxSize,
 					partnerChainInboxSize: 1,
 				});
 
-			const result = await command.verify(commandVerifyContext);
+			const result = await command.verify(commandExecuteContext);
```

Comment:
we need to call execute after we remove it from verify function. Also, add an assertion if  InvalidRMTVerification was called or not